api/src/main/java/org/openmrs/module/kenyaemr/chore/MigratePatientContactPersonAttributes.java

```diff
@@ -0,0 +1,186 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.kenyaemr.chore;
+
+import org.openmrs.api.context.Context;
+import org.openmrs.module.kenyacore.chore.AbstractChore;
+import org.springframework.stereotype.Component;
+
+import java.io.PrintWriter;
+
+/**
+ * Migrate Patient contacts from kenyaemr_hiv_testing_patient_contact
+ */
+@Component("kenyaemr.chore.MigratePatientContactPersonAttributes")
+public class MigratePatientContactPersonAttributes extends AbstractChore {
+
+    /**
+     * @see AbstractChore#perform(PrintWriter)
+     */
+
+    @Override
+    public void perform(PrintWriter out) {
+
+        String insertPersonAttributes = "insert into person_attribute (person_id, value, person_attribute_type_id, creator, date_created, changed_by,\n" +
+                "                              date_changed, voided, voided_by, date_voided, void_reason, uuid)\n" +
```

Comment:
This looks like a complex query. Wondering if using the module service to get contacts would have simplified the logic